### PR TITLE
Fix signing pattern to capture all winmd and dll in the transport package

### PR DESCRIPTION
**Pull request by kythant**
Fix signing pattern to capture all winmd and dll in the transport package

### CODE REVIEW

File: build/ProjectReunion-BuildFoundation.yml [hunk 1]
```diff
@@ -194,9 +194,8 @@ jobs:
 - job: SignBinariesAndPublishSymbols
   variables:
     signingPattern: |
-        lib\uap10.0\*.winmd
-        lib\net5.0-windows\*.dll
-        runtimes\*\*\*.dll
+        **/*.winmd
+        **/*.dll
```
Comment (DrusTheAxe) [inline build/ProjectReunion-BuildFoundation.yml:198]: Also needs `**/*.exe*` ?

**Review by DrusTheAxe — APPROVED** [review]

File: build/ProjectReunion-BuildFoundation.yml (hunk 1 above)
Comment (kythant) [reply build/ProjectReunion-BuildFoundation.yml:198]: Yea

**Pull request merged**